packages/block-library/src/button/block.json

```diff
@@ -95,14 +95,6 @@
 				"//": "100% causes an oval, but any explicit but really high value retains the pill shape.",
 				"radius": "9999px"
 			},
-			"color": {
-				"text": "#fff",
-				"background": "#32373c"
-			},
-			"typography": {
-				"fontSize": "1.125em",
-				"textDecoration": "none"
-			},
```

Comment:
Can you clarify again why we're moving these styles to Core's `theme.json`?

Reply:
Yes, we need these styles to be shared across all elements, so there's nowhere else to put them.